shop.json

```diff
@@ -0,0 +1,8 @@
+{
+  "items": [
+    {"name": "Dagger", "description": "Kill one player", "price": "750", "emoji": ":dagger:"},
+    {"name": "Randium Face Reveal", "description": "Randium will send a picture of his face", "price": "9999999999", "emoji": ":grinning_face_with_big_eyes:"}
```

Comment:
Why are the prices in string format?

Reply:
They're only used for string concatentation

Reply:
Why would you need that? This is Python, you don't need to format ints as strings so you can put them together.

Reply:
Changed 👍